src/components/Stepper/src/Stepper.vue

```diff
@@ -13,9 +13,33 @@
 		>
 			<minus :class="$s.Icon" />
 		</m-button>
-		<span :class="$s.Quantity">
-			{{ value }}
-		</span>
+		<div
+			:class="$s.Quantity"
+		>
+			<input
+				v-if="isSettingManualValue"
+				ref="manualInput"
+				v-model="manualValue"
+				:class="$s.QuantityManualInput"
+				:min="min"
+				:max="max"
+				type="number"
+				inputmode="numeric"
+				align="center"
+				@blur="commitManualValue"
+				@keyup.enter="commitManualValue"
```

Comment:
Pro / cons either way, but removing the `enter` on this would catch invalid data on each keyup:
![CleanShot 2022-05-06 at 12 29 27](https://user-images.githubusercontent.com/129122/167205291-5db170eb-4f8c-4c23-8ab8-6145db922faa.png)

Reply:
I think I'm going to switch over the `blur` and `keyup.enter` since it seems to mimic this behaviour anyway.